cves/kernel/CVE-2016-0823.yml

```
@@ -322,8 +333,10 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The fixing commit 1c90308e7a77af6742a97d1021cca923b23b7f0d is signed off by
+    Konstantin Khlebnikov.
```

I looked this up and saw that the following were the people who worked on it. You just pasted the wrong person as the sign off but I think you should include the others. Just because the original person also felt like they were important enough to cc them. 

```suggestion
  note: |
Signed-off-by: Kirill A. Shutemov <kirill.shutemov@linux.intel.com>
Acked-by: Konstantin Khlebnikov <khlebnikov@openvz.org>
Acked-by: Andy Lutomirski <luto@amacapital.net>
Cc: Pavel Emelyanov <xemul@parallels.com>
Cc: Andrew Morton <akpm@linux-foundation.org>
Cc: Mark Seaborn <mseaborn@chromium.org>
Cc: stable@vger.kernel.org
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
```